### PR TITLE
Need to use set_balance_in_account_currency

### PR DESCRIPTION
**Pull request by revant**
https://github.com/revant/drop_ship/commit/b9ad5178e4f75a73be9f7b36a1b3d691c8937746
@nabinhait 
needed to make GL Entry

### CODE REVIEW

File: erpnext/controllers/accounts_controller.py [hunk 1]
```diff
@@ -541,3 +524,20 @@ def _on_previous_row_error(row_range):
 			_on_previous_row_error("1 - %d" % (tax.row_id,))
 		elif tax.get("category") == "Valuation":
 			frappe.throw(_("Valuation type charges can not marked as Inclusive"))
+
+def set_balance_in_account_currency(gl_dict, conversion_rate=None, company_currency=None, account_currency=None):
```
Comment (nabinhait) [inline erpnext/controllers/accounts_controller.py]: Don't change the position of the argument "account_currency", keep it in the second place.

Comment (nabinhait) [inline erpnext/controllers/accounts_controller.py]: You should also change wherever it is called in erpnext.

**Comment by revant** [comment]
I searched set_balance_in_account_currency in all files, found it used only in accounts_controller.py, that I changed.
Are arguments correct?

**Comment by nabinhait** [comment]
@revant I have fixed an issue on your PR, it gives an error for stock entry. Check https://github.com/frappe/erpnext/commit/67f40775ff588ea997522ebb18028e2a3b09e6cf

**Pull request merged**